Flight Logs

naimishemt opened the pull request:
Please verify the logs.

naimishemt commented:
commit

naimishemt commented:
We have submitted the logs . Kindly verify

naimishemt commented:
kindly verify the logs

sourabhsaini11 commented:
@naimishemt, please get the below issues fixed

sourabhsaini11 commented:
**NOTE**: Above issues will not be considered as holistic as the Api-contract wasn't implemented completely.
Also, please provide rest of the flows as well i.e. 
Flow-1 (Happy quantity 1)
Flow-2 (Happy Adult x2 child x1)
Flow-3 (cancellation flow)

naimishemt commented:
log submitted .

naimishemt commented:
Metro RSF Log Submitted

sourabhsaini11 commented:
@naimishemt, please get the below issues fixed as well
common_issue:
- tax_id is missing at billing
- message.order.quote.breakup[0].price.currency is missing in the quote objects
- timestamp must have seconds & milliseconds present in it while milliseconds being precise upto 3 places only
- select_2 api pair calls are missing

search:
- message.intent.fulfillment.stops[0].time.timestamp format is not correct according to the documentation

on_search:
- context.bpp_id and context.bpp_uri is missing
- INFO tag-group is missing
- selected is not required at items[1].quantity
- additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items

select: 
- descriptor object not required in message.order.items[0].descriptor
- parent_item_id should map to id of parent item object at items[1]

on_select
- INFO, VEHICLE_AVAIBALITY & VEHICLE_GRID atg-group should be present instead of FARE_TYPE at fulfillments
- add_ons should not be part of parent item object, should be aprt of child objects instead
- additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items
- breakup.title should be fixed enum values as defined in api-contract
- fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing

init:
- vehicle object is missing in message.order.fulfillments
- message.order.items[0].descriptor object is not allowed when comapred with documentation

on_init
- INFO tag-group should be present instead of FARE_TYPE at fulfillments
- additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items
- breakup.title should be fixed enum values as defined in api-contract
- fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing

confirm: 
- message.order.id mismatch between confirm and on_confirm
- type is not required at fulfillments[0]

on_confirm:
- message.order_id is not required when compared to documantation
- INFO tag-group should be present instead of FARE_TYPE at fulfillments
- fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing
- add_ons should not be empty at items
- additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items
- tag SETTLEMENT_WINDOW is missing in SETTLEMENT_TERMS tag-group
- breakup.title should be fixed enum values as defined in api-contract

on_cancel:
- INFO tag-group should be present instead of FARE_TYPE at fulfillments
- fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing
- add_ons should not be empty at items
- additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items
- tag SETTLEMENT_WINDOW is missing in SETTLEMENT_TERMS tag-group
- breakup.title should be fixed enum values as defined in api-contract
- location, type, time are missing in fulfillments.stops

Also, kindly segregate cancel calls in a separate flow & provide IGM flow.

naimishemt commented:
on_search:


INFO tag-group is missing-- Info tag data not availabel
selected is not required at items[1].quantity--- Requeried
additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items: Required

select:

descriptor object not required in message.order.items[0].descriptor-- Required
parent_item_id should map to id of parent item object at items[1]-- Discussion

on_select

INFO, VEHICLE_AVAIBALITY & VEHICLE_GRID atg-group should be present instead of FARE_TYPE at fulfillments --- INFO, VEHICLE_AVAIBALITY & VEHICLE_GRID, we have not data  
add_ons should not be part of parent item object, should be aprt of child objects instead-- Discussion
additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items--- Requeried
breakup.title should be fixed enum values as defined in api-contract
fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing--- We dont have data yet.

init:


message.order.items[0].descriptor object is not allowed when comapred with documentation-- Reqeuired

on_init

INFO tag-group should be present instead of FARE_TYPE at fulfillments-- We dont have data for it.
additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items-- Reqeuired
breakup.title should be fixed enum values as defined in api-contract-- Discussion
fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing-- We dont have data  yet

confirm:

message.order.id mismatch between confirm and on_confirm-- this order will not match with On_Confirm order id.


on_confirm:

message.order_id is not required when compared to documantation--- Reqeuired
INFO tag-group should be present instead of FARE_TYPE at fulfillments---Data dont have yet
fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing---Data dont have yet 
additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items-- Requeried
breakup.title should be fixed enum values as defined in api-contract-- Discussion

on_cancel:

INFO tag-group should be present instead of FARE_TYPE at fulfillments--- Dont have data yet
fulfillment object representing SEAT_GRID tag-group & "type": "TICKET" is missing--- Dont have data yet
additional tag-groups JOURNEY_TIME, TAX_BREAK_UP present at items--Reqeuired
breakup.title should be fixed enum values as defined in api-Discussion

naimishemt commented:
Commit Logs 1 Happy Traveler

naimishemt commented:
Commit 2 Adult and 1 Child Logs

sourabhsainii commented:
@naimishemt, please get the below issues fixed
common_issues
- timestamp must be in UTC(YYYY-MM-DDTHH:MN:SS.MSSZ) format
- title should be BASE_FARE instead of BASIC_FARE at quote.breakup, quantity should also be present
- price precision should be upto 2 decimal only
- item.id is getting changed from select to confirm calls
- FARE_BREAK_UP tag-group & quote.breakup are not matching
- additional tags REFUNDABLE & ROUND_TRIP_FARE present in FARE_BREAK_UP tag-group
- code should be BASE_FARE instead of BASIC_FARE at FARE_BREAK_UP tag-group

on_search
- tag PARENT_ID is missing in INFO tag-group
- descriptor.code is missing in INFO tag-group
- INFO tag-group is missing for fulfillments with type as 'TRIP'
- For items.add_ons provide appropriate price &  description.name
- remove attributes maximum & minimum from quantity
- attribute 'available' should be used instead of 'selected' at items.add_ons
- refund_terms is missing in items

select
- duplicate fulfillments present
- items array should contain one object i.e. parent item with id as '1-O0-1-0'& 2 child objects referring to parent_item_id '1-O0-1-0'

on_select
- INFO tag-group is missing at fulfillments.stops
- FARE_TYPE tag-groups is not required in fulfillments
- items object with parent_item_id is missing
- add_ons should not be present if not selected

init
- mutiple fulfillments based on adult & child tickets (ticket holder details) are missing
- status must be NOT-PAID at payments
- items array should contain one object i.e. parent item with id as '1-O348-1'& 2 child objects referring to parent_item_id '1-O348-1'

on_init
- INFO tag-group is missing at fulfillments.stops
- FARE_TYPE tag-groups is not required in fulfillments
- items object with parent_item_id is missing
- items with descriptor CHILD_TICKET is missing, only ADULT_TICKET object is present.
- add_ons should not be present if not selected

confirm
- id should be created on /on_confirm
- mutiple fulfillments based on adult & child tickets (ticket holder details) are missing
- items array should contain one object i.e. parent item with id as '1-O350-1'& 2 child objects referring to parent_item_id '1-O350-1'
- payments.id can be creared on /on_confirm

on_confirm
- INFO tag-group is missing at fulfillments.stops
- FARE_TYPE tag-groups is not required in fulfillments
- add_ons should not be present if not selected
- selected.count must be 2 for ADULT_TICKET at items
- items object with parent_item_id is missing

on_cancel
- INFO tag-group is missing at fulfillments.stops
- FARE_TYPE tag-groups is not required in fulfillments
- items object with code as CHILD_TICKET is missing
- selected.count must be 2 for ADULT_TICKET at items
- items object with parent_item_id is missing
- tags SETTLEMENT_BASIS, COURT_JURISDICTION & MANDATORY_ARBITRATION are missing at SETTLEMENT_TERMS tag-group
- params.amount is changed

on_status
- remove fulfillments array present in provider
- INFO tag-group is missing at fulfillments.stops
- FARE_TYPE tag-groups is not required in fulfillments
- items object with code as CHILD_TICKET is missing
- selected.count must be 2 for ADULT_TICKET at items
- type is missing in items
- stops is incomplete
- items object with parent_item_id is missing
- tags SETTLEMENT_BASIS, COURT_JURISDICTION & MANDATORY_ARBITRATION are missing at SETTLEMENT_TERMS tag-group
- tags should not be empty at provider

naimishemt commented:
Kinldy verify the logs.

timestamp must be in UTC(YYYY-MM-DDTHH:MN:SS.MSSZ) format--- Fixed
title should be BASE_FARE instead of BASIC_FARE at quote.breakup, quantity should also be present-FIXED
price precision should be upto 2 decimal only-- FIXED
item.id is getting changed from select to confirm calls--not valid
FARE_BREAK_UP tag-group & quote.breakup are not matching--- FARE_BREAK_UP Removed
additional tags REFUNDABLE & --- Fixed
ROUND_TRIP_FARE present in FARE_BREAK_UP tag-group-- ROUND_TRIP_FARE Requeired ,need to dicsuss in which tag we can add.
code should be BASE_FARE instead of BASIC_FARE at FARE_BREAK_UP tag-group-- Fixed

on_search

tag PARENT_ID is missing in INFO tag-group--- Fixed
descriptor.code is missing in INFO tag-group--FIXED
INFO tag-group is missing for fulfillments with type as 'TRIP'
For items.add_ons provide appropriate price & description.name--- FIXED
remove attributes maximum & minimum from quantity-- FIXED
attribute 'available' should be used instead of 'selected' at items.add_ons--FIXED
refund_terms is missing in items

select

duplicate fulfillments present-- FIXED


on_select

INFO tag-group is missing at fulfillments.stops--- Data Issue
FARE_TYPE tag-groups is not required in fulfillments--fixed
items object with parent_item_id is missing--- FIXED
add_ons should not be present if not selected--- FIXED

init

mutiple fulfillments based on adult & child tickets (ticket holder details) are missing--- Fixed
status must be NOT-PAID at payments-- FIXED
items array should contain one object i.e. parent item with id as '1-O348-1'& 2 child objects referring to parent_item_id '1-O348-1'-- FIXED

on_init

INFO tag-group is missing at fulfillments.stops-- Data issue
FARE_TYPE tag-groups is not required in fulfillments---Fixed
items object with parent_item_id is missing
add_ons should not be present if not selected-- FIXED

confirm

id should be created on /on_confirm---  Keep as per discussion
mutiple fulfillments based on adult & child tickets (ticket holder details) are missing--- Fixed
items array should contain one object i.e. parent item with id as '1-O350-1'& 2 child objects referring to parent_item_id '1-O350-1'-- Fixed
payments.id can be creared on /on_confirm--- Keep as per discussion

on_confirm

INFO tag-group is missing at fulfillments.stops-- Data issue
FARE_TYPE tag-groups is not required in fulfillments--- Fixed
add_ons should not be present if not selected--- FIXED
selected.count must be 2 for ADULT_TICKET at items--- Fixed
items object with parent_item_id is missing--- FIXED

on_cancel

INFO tag-group is missing at fulfillments.stops--- Data
FARE_TYPE tag-groups is not required in fulfillments-- fixed
items object with code as CHILD_TICKET is missing---FIXED
selected.count must be 2 for ADULT_TICKET at items-- Fixed 
items object with parent_item_id is missing-- Umesh
tags SETTLEMENT_BASIS, COURT_JURISDICTION & MANDATORY_ARBITRATION are missing at SETTLEMENT_TERMS tag-group
params.amount is changed

on_status

remove fulfillments array present in provider---FIXED
INFO tag-group is missing at fulfillments.stops-- Data
FARE_TYPE tag-groups is not required in fulfillments--- FIXED
items object with code as CHILD_TICKET is missing--FIXED
selected.count must be 2 for ADULT_TICKET at items-- FIXED
type is missing in fulfillments--FIXED
stops is incomplete--FIXED
items object with parent_item_id is missing--FIXED
tags SETTLEMENT_BASIS, COURT_JURISDICTION & MANDATORY_ARBITRATION are missing at SETTLEMENT_TERMS tag-group-- 
tags should not be empty at provider--- FIXED

sourabhsainii commented:
@naimishemt, please get the below issues fixed
common_issues
- bap_uri shouldn't be localhost
- tax breakup is incorrect, at quote

on_search
- add_ons price must be reasonable
- for fulfillment.type 'CONNECT', attributes stops & vehicle are missing
- cancellation_terms is missing
- tag CABIN_BAGGAGE is missing in GENERAL_INFO tag-group
- use attribute refund_terms instead of tag REFUNDABLE at FARE_BREAK_UP tag-group

on_select
- refund_terms is missing in items
- additional tag-group FARE_BREAK_UP present in items
- tag-group FARE_TYPE is missing in items
- add_ons should not be present if not selected by user
- fulfillments object having id & type TICKET are missing

on_init
- billing is missing
- fulfillments objects having customer details with "type": "TICKET" are missing
- add_ons should not be present if not selected by user
- refund_terms is missing in items
- id is missing for child items object
- additional tag-group FARE_BREAK_UP present in items
- tag-group FARE_TYPE is missing in items
- quantity is missing for id's: '1-O3276-1'(flow-1) '1-O3923-1' & '1-O3923-2'(flow-2), at quote.breakup

confirm
- id is missing for fulfillment objects having customer details 

on_confirm
- billing is missing
- valid to must be in UTC(YYYY-MM-DDTHH:MN:SS.MSSZ) format, at fulfillments.authorization
- fulfillments objects having customer details with "type": "TICKET" are missing
- add_ons should not be present if not selected by user
- refund_terms is missing in items
- id is missing for child items object
- additional tag-group FARE_BREAK_UP present in items
- tag-group FARE_TYPE is missing in items
- quantity is missing for id's: '1-O3274-1'(flow-1) '1-O3921-1' & '1-O3921-2'(flow-2), at quote.breakup

on_cancel
- billing is missing
- fulfillments objects having customer details with "type": "TICKET" are missing
- refund_terms is missing in items
- child items object referring to parent_item_id is missing
- additional tag-group FARE_BREAK_UP present in items
- tag-group FARE_TYPE is missing in items
- quantity is missing for id's: '1-O3276-1'(flow-1) '1-O3923-1' & '1-O3923-2'(flow-2), at quote.breakup
- multiple SETTLEMENT_TERMS tag-groups present in payments
- status is missing
- cancellation_fee.percentage is changed from what was sent in /on_confirm, at cancellation_terms
- Tag BUYER_FINDER_FEES_PERCENTAGE value is changed from what was sent in /on_confirm, at cancellation_terms

on_status
- fulfillments objects having customer details with "type": "TICKET" are missing
- refund_terms is missing in items
- child items object referring to parent_item_id is missing (flow-1 only)
- id is missing for child items object (flow-2 only)
- additional tag-group FARE_BREAK_UP present in items
- tag-group FARE_TYPE is missing in items
- quantity is missing for id's: '1-O3276-1'(flow-1) '1-O3923-1' & '1-O3923-2'(flow-2), at quote.breakup
- multiple SETTLEMENT_TERMS tag-groups present in payments
- cancellation_fee.percentage is changed from what was sent in /on_confirm, at cancellation_terms
- Tag BUYER_FINDER_FEES_PERCENTAGE value is changed from what was sent in /on_confirm, at cancellation_terms

naimishemt commented:
All ponits are fixed ..

tax breakup is incorrect, at quote-- Discussion

on_search
for fulfillment.type 'CONNECT', attributes stops & vehicle are missing--- Discussion


on_select

fulfillments object having id & type TICKET are missing--- Discussion

sourabhsaini11 commented:
@naimishemt, please get the below issues fixed as well
common_issues
- tax breakup is incorrect, at quote
- tag SETTLEMENT_AMOUNT value is getting changed between /action & /on_action calls

on_search
- for fulfillment.type 'CONNECT', attributes stops & vehicle are missing
- add_ons price must be reasonable

on_select
- fulfillments object having id & type TICKET are missing
- additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
- multiple GENERAL_INFO tag-groups are present

init
- additional attribute type send in fulfillments

on_init
- additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
- multiple GENERAL_INFO tag-groups are present

confirm
- additional attribute type send in fulfillments
- person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)

on_confirm
- fulfillments should have only 1 object capturing customer info instead of 2. (1 Ticket is selected as per flow-1)
- person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)
- additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
- multiple GENERAL_INFO tag-groups are present

on_cancel
- additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
- multiple GENERAL_INFO tag-groups are present
- quote.price is increased upon cancellation

on_status
- additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
- multiple GENERAL_INFO tag-groups are present
- billing details got changed

naimishemt commented:
tax breakup is incorrect, at quote--- Json Document Change From ONDC SIDE

on_search

for fulfillment.type 'CONNECT', attributes stops & vehicle are missing --- Discussion
add_ons price must be reasonable --- JSON CONTRACT WRONG OF ONDC

on_select

fulfillments object having id & type TICKET are missing--- Once Seat Map Implemented, Then it will be added
additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group --- ONDC Team will update , In which tag need to add this value
multiple GENERAL_INFO tag-groups are present
init

additional attribute type send in fulfillments--- Discussed in meeting this is not an Issue

on_init

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group--  ONDC Team will update , In which tag need to add this value.
multiple GENERAL_INFO tag-groups are present

confirm

additional attribute type send in fulfillments--- Discussed in meeting this is not an Issue
person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)-- After discussion We added this tag.Now We are chaning flow after meeting discussion

on_confirm

fulfillments should have only 1 object capturing customer info instead of 2. (1 Ticket is selected as per flow-1)-- After discussion We added this tag.Now We are chaning flow after meeting discussion
person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)---- -- After discussion We added this tag.Now We are chaning flow after meeting discussion
additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group--   ONDC Team will update , In which tag need to add this value.
multiple GENERAL_INFO tag-groups are present

on_cancel

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group--  ONDC Team will update , In which tag need to add this value.
multiple GENERAL_INFO tag-groups are present
quote.price is increased upon cancellation

on_status

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group--   ONDC Team will update , In which tag need to add this value.
multiple GENERAL_INFO tag-groups are present
billing details got changed

naimishemt commented:
Post discussion, below are the update from EMT :

tax breakup is incorrect, at quote
[EMT ] :ONDC Changed structure yesterday and since not update so EMT was sitting on earlier structure, implementing again

on_search

for fulfillment.type 'CONNECT', attributes stops & vehicle are missing 
[EMT] As per discussion, Sourabh will discuss internally and will let emt know for next step.

add_ons price must be reasonable 
[EMT] This needs to be updated from ONDC,

on_select

fulfillments object having id & type TICKET are missing
[EMT] As discussed, this is related with Seat Map, which is not in place to begin with as many of the airline do not support it online.
additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group 
[EMT] As discussed, please share relevant tag name to put this information, ONDC Team to check and update.
multiple GENERAL_INFO tag-groups are present
[EMT] EMT team to fix it

init

additional attribute type send in fulfillments
[EMT] As discussed and confirmed, its not an issue and will be taken out from here as a issue pointer.

on_init

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
[EMT] As discussed, ONDC team will share tag information where we need to put this data.

multiple GENERAL_INFO tag-groups are present
[EMT] EMT team to fix it

confirm

additional attribute type send in fulfillments
[EMT] As discussed and confirmed, its not an issue and this item will be taken out from here

person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)
[EMT] This was in place as per last discussed, but today new approach has been suggested which EMT team is implementing

on_confirm

fulfillments should have only 1 object capturing customer info instead of 2. (1 Ticket is selected as per flow-1)
[EMT] This was done,as per suggestions from last call, now post todays call updating again as per suggestions\

person.name should only consist name of the person, remove additional info at fulfillments[4](Flow-2 only)
[EMT] This was done,as per suggestions from last call, now post todays call updating again as per suggestions\

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
[EMT] As discussed, ONDC Team needs to provide a tag where to put in this information

multiple GENERAL_INFO tag-groups are present
[EMT] EMT team to fix it

on_cancel

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
[EMT] As discussed, ONDC Team needs to provide a tag where to put in this information

multiple GENERAL_INFO tag-groups are present
[EMT] EMT to fix
quote.price is increased upon cancellation
[EMT] EMT team needs to check and fix

on_status

additional tag ROUND_TRIP_FARE present in GENERAL_INFO tag-group
[EMT] As discussed, ONDC Team needs to provide a tag where to put in this information

multiple GENERAL_INFO tag-groups are present
[EMT] EMT to fix

billing details got changed
[EMT] EMT to fix and update

naimishemt commented:
Logs are updated .Please verify . 
Few points we will fixed,once we will get the update from ONDC team.

naimishemt commented:
@92shreyansh  Yes, This PR active so please verify this.

sourabhsaini11 commented:
@naimishemt, please get the below issues fixed
common_issues
- tag SETTLEMENT_AMOUNT value is getting changed between /action & /on_action calls (flow-2 only)

on_search
- for fulfillment.type 'CONNECT', attributes stops & vehicle are missing

on_select
- fulfillments object having id & type TICKET are missing
- multiple tags ROUND_TRIP_FARE found in FARE_BREAK_UP tag-group at items[0] (flow-2 only)

on_cancel
- refund_terms & time is missing in items

on_status
- refund_terms & time is missing in items
- bank_account_number, bank_account_number & virtual_payment_address are missing in payments.params

sourabhsaini11 commented:
@naimishemt, below are the updated issues post-discussion
common_issues
- item.id should remain same post /init
- remove tag ROUND_TRIP_FARE from tag-group FARE_BREAK_UP  
- tag SETTLEMENT_AMOUNT value is getting changed between /action & /on_action calls (flow-2 only)

on_select
- fulfillments object consisting of id & type TICKET is missing

on_cancel
- refund_terms & time is missing in items

on_status
- refund_terms & time is missing in items
- bank_account_number, bank_code & virtual_payment_address are missing in payments.params

naimishemt commented:
Log Submitted

92shreyansh commented:
Transaction Logs are Approved.

92shreyansh commented:
Please update the PR so that only TRV12 transaction logs get merged and have a separate PR for RSF

Post this will merge the PR

naimishemt commented:
Log updated. KIndly verify

naimishemt commented:
Log Commited

naimishemt commented:
Commit

naimishemt commented:
Commit Hotel Logs

naimishemt commented:
Flight IGM log Sumitted

naimishemt commented:
Metro IGM Logs Sumitted.

RupalSingla commented:
IGM:

issue close:

created_at and updated_at cannot be same .

@naimishemt

naimishemt commented:
IGM logs updated.

amitsinha07 commented:
Hii @naimishemt 
Feedbacks for IGM Logs

1. **issue:**
   - Missing required property 'person' in `/message/issue/issue_actions/complainant_actions/0/updated_by`.
   - Organization's Name mismatch with BAP ID in `/issue` API.
   - Organization domain mismatch with context domain.
   - `created_at`, `updated_at`, and `complainant_action[0]/updated_at` should be in ISO format. (Context timestamp is correct)

2. **issue_CLOSE:**
   - Missing required property 'person' in `/message/issue/issue_actions/complainant_actions/0/updated_by` and `/message/issue/issue_actions/complainant_actions/1/updated_by`.
   - Phone Number not in the valid range for `/issue_close` API.
   - Organization's Name mismatch with BAP ID.
   - Organization domain mismatch with context domain.

3. **on_issue:**
   - Message ID mismatch in `/issue` and `/on_issue`.

4. **on_issue_status:**
   - Transaction ID mismatch in `/issue` and `/on_issue_status`.
   - Message ID mismatch in `/issue_status` and `/on_issue_status`.
   - `updated_at` should be greater than `created_at`.
   - `updated_at` in `message/issue/issue_action/respondent_actions` should not exceed `updated_at` in `on_issue_status`.
   - `respondent_action[0]/updated_at` is not matching with `respondent_action[0]/updated_at` of on_issue call. 

5. **on_issue_status_unsolicited:**
   - Missing required property 'short_desc' in `/message/issue/issue_actions/respondent_actions/1`.
   - `respondent_action[0]/updated_at` is not matching with `respondent_action[0]/updated_at` of `on_issue_status` call.

RupalSingla commented:
Issue Close :
created_at and updated_at cannot have same values.
@naimishemt

mishra-umesh commented:
Hi team Logs submitted again .Kindly verify

JyotiSharma0711 commented:
Easemytrip(RSF)

"receiver_recon": 
  
"/context/core_version must be equal to one of the allowed values (1.0.0)",
"/message/orderbook/orders must NOT have fewer than 5 items",
"/message/orderbook/orders/0 must have required property 'withholding_tax_gst'",
"/message/orderbook/orders/0 must have required property 'withholding_tax_tds'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/collected_by_status'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/withholding_amount'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/withholding_amount_status'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/return_window'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/return_window_status'",
"/message/orderbook/orders/0/payment/uri must match format \"uri\"",
"/message/orderbook/orders/0/payment/@ondc~1org~1settlement_window must match format \"duration\"",
"/message/orderbook/orders/0/payment/@ondc~1org~1settlement_details/0/settlement_ifsc_code must match pattern \"^[A-Z]{4}0[A-Z0-9]{6}$\"",
"respTime": "context timestamp should be greater than created_at"

"on_receiver_recon": 

orders.order_recon_status is a string
orders.invoice_no missing
orders.message missing

RupalSingla commented:
"receiver_recon": 
  
"/context/core_version must be equal to one of the allowed values (1.0.0)",
"/message/orderbook/orders must NOT have fewer than 5 items",
"/message/orderbook/orders/0 must have required property 'withholding_tax_gst'",
"/message/orderbook/orders/0 must have required property 'withholding_tax_tds'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/collected_by_status'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/withholding_amount'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/withholding_amount_status'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/return_window'",
"/message/orderbook/orders/0/payment must have required property '@ondc/org/return_window_status'",
"/message/orderbook/orders/0/payment/uri must match format \"uri\"",
"/message/orderbook/orders/0/payment/@ondc~1org~1settlement_window must match format \"duration\"",
"/message/orderbook/orders/0/payment/@ondc~1org~1settlement_details/0/settlement_ifsc_code must match pattern \"^[A-Z]{4}0[A-Z0-9]{6}$\"",
"respTime": "context timestamp should be greater than created_at"

"on_receiver_recon": 

orders.order_recon_status is a string
orders.invoice_no missing
orders.message missing

@mishra-umesh

JyotiSharma0711 commented:
Reminder:

Kindly note that this PR is scheduled to be closed within the next 1 to 2 weeks. Please ensure that any required changes from your side are addressed within this timeframe.

JyotiSharma0711 commented:
Based on the last update, we are proceeding to close this PR. Thank you for your time and support
